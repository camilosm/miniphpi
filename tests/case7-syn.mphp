$a = read "entre com um numero: ";
if ($a < 0) {
  $b = "menor";
} else if ($a > 0) {
  $b = "maior";
} else {
  $b = igual";
}
echo $b;
