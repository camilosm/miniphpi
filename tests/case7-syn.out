04: Lexema n√£o esperado [if]
